vendor/github.com/hyperledger/fabric-protos-go/orderer/cluster.pb.go

```diff
@@ -30,6 +31,7 @@ type StepRequest struct {
 	// Types that are valid to be assigned to Payload:
 	//	*StepRequest_ConsensusRequest
 	//	*StepRequest_SubmitRequest
+	//	*StepRequest_AuthRequest
```

Comment:
we need to make a different gRPC service, not extend an existing one.

Reply:
A new service with only the authentication method?

Reply:
@yacovm  opened a PR https://github.com/Fabric-orderer-collaboration/fabric-protos/pull/2 for new service